### ISSUE TITLE
Server: New Feature : couloured tabbed categories like in OneNote

### ISSUE THREAD

**Issue by ymm4** [Fmstrat/ownnote#221]
Hello.

A great feature would be coloured tabbed categories like in Microsoft OneNote:
* Categories in colored tabs on a horizontal top line.
* Notes belonging to the selected category on a left vertical column.
This brings viewing into 2 dimensions instead of one, and it greatly helps navigation, especially on notes systems with many notes.
![sans titre](https://cloud.githubusercontent.com/assets/13817654/9292781/5e990388-440b-11e5-85d3-c6dcd44cee02.png)

A less desirable alternative is to display hierarchies on 3 vertical panes : categories / notes names / notes contents, like here:
![sans titre](https://cloud.githubusercontent.com/assets/13817654/9292779/39ee09d4-440b-11e5-815d-719407ae08aa.png)

Best regards to all.

**Comment by Fmstrat** [comment]
I'll have to think on this. Keeping things consistent between web and android is important, and ownCloud has a design style-guide from a navigation standpoint that I am trying to stay within, too.

**Comment by ymm4** [comment]
I believe that the 2 dimensional aspect is hugely important.
If OwnNote stays one-dimensional, then it wil be used only dor minor stuff.Only a few dozen notes max can be managed conveniently.
Thanks to the 2 dimensional aspect of OneNote, I am able to easy manage hundreds of notes. It has become the data center of my personal and professional life (in separate notebooks).
However, I am uncomfortable at letting Microsoft own my data. My dream is to be able to manage all these notes on Owncloud, with open source software.
OwnNote could be it ...

**Comment by emtie** [comment]
Yes 2 dimensional would be really great. I'm waiting for this since the first version. ownNote is great and that would be the cherry on the cake ;-)

thanks emtie

**Comment by ThomasDaheim** [comment]
+1 looks great with tabs & colors!
